lib/util.mli

```diff
@@ -85,6 +85,13 @@ val member : string -> t -> t
       object [obj], or [`Null] if [k] is not present in [obj].
       @raise Type_error if [obj] is not a JSON object. *)
 
+val path : string list -> t -> t
+(* [path l obj] returns the value associated with the list of keys [l] in
+     the JSON object [obj], or [`Null] if [l] did no led to an object in
+     [obj]. [path [k1; ...; kn] obj] is the same as
+     [member kn (... (member k1 obj))]
+     @raise Type_error if [obj] is not a JSON object. *)
```

Comment:
I think in 2022 we shouldn't be adding more unsafe functions that throw exceptions. It would be better to return `t option` here and I am somewhat unsure that returning `` `Null`` is a behavior that makes sense in OCaml. If I saw a function like this I would not expect `path ["missing"] (`Assoc [])` to return `` `Null``, I would rather expect it to throw `Not_found` following the stdlib.

(I am aware that it uses `member` for which the exact same concerns apply which unfairly gets a free pass for being like this since forever)